conf/params.config

```diff
@@ -20,18 +20,18 @@ params {
 
     // Max resource options
     // Defaults only, expecting to be overwritten
-    max_memory                 = '128.GB'
-    max_cpus                   = 16
-    max_time                   = '240.h'
+    max_memory                 = '16.GB'
```

Comment:
These seem too low, the maximums are enforced even if a label states the process should have higher resource so no individual process will ever be allowed to have more memory / cpu than what is stated here (unless higher is specified via params).

Reply:
Also worth noting, this will only ever work with a cloud executor, local executor is an exceedingly bad neighbour and will hoover up as much resource as it wants

Reply:
How do you mean? I don't see any reason why this couldn't be run on a laptop. It's not going to have the best performance but it could run?

Reply:
I meant that the max resource args only work with cloud executor not the pipe generally since the local executor doesn't care about the process cpus / memory defs outside of where they're accessed like `$task.cpus`

Reply:
I definitely needed to adjust the caps to get it to work on my laptop. I've added a commit to make local running have lower resources with a flag

Reply:
In that case I may well be chatting nonsense hah